FOR REVIEW: Adding Endgame stuff (#335)
makerdao/governance-manual: delegation/alignment-conservers.md

# Alignment Conservers

{% hint style="warning" %}
This documentation describes functionality and processes that MakerDAO is in the process of implementing. Be aware that parts may be inaccurate or out of date.
{% endhint %}

> An Alignment Conserver (CC) is a guardian of the [Maker Atlas](../governance/atlas.md). They must preserve the Atlas and oppose any entities threatening it.

Reviewer: ```suggestion
An Alignment Conserver (AC) is a guardian of the [Maker Atlas](../governance/atlas.md). They must preserve the Atlas and oppose any entities threatening it.
```

## Types

> There are two types of Alignment Conservers, an [AVC](avc.md) Member and an [Aligned Delegate](aligned-delegates.md).

Reviewer: ```suggestion
There are two types of Alignment Conservers: [AVC](avc.md) Members and [Aligned Delegates](aligned-delegates.md).
```

> The Arbitration Scope gives recognition to each type of Alignment Conserver if they meet the requirements. Ecosystem Actors that become recognized as a type of Alignment Conserver may not assume any other ecosystem role, including that of a different type of Constitutional Conserver.

Reviewer: Very confusing

## Purpose

The Maker Atlas is designed to safeguard the interests of MKR holders. By preserving and protecting the Atlas from changes, Alignment Conservers function as guardians of MKR holders' interests.

Alignment Conservers may never change the Atlas or governance dynamics unless they do it within the Atlas processes and frameworks. During Pregame, AVC Members are allowed and encouraged to help refine the Maker Atlas.

> ## Eligibility requirements

Reviewer: ```suggestion
## Eligibility Requirements
```

To be eligible, Alignment Conservers must make a public pledge committing to:
- Preserve the Atlas
- Oppose and publicly expose all forms of corruption, organizational drifts, and other threats to the Atlas

Alignment Conservers must be anonymous. During Pregame, anonymity is encouraged but not required.

> An Alignment Conserver can be **derecognized** if it acts against the pledge or against any of the requirements attached to their type.

Reviewer: ```suggestion
An Alignment Conserver can be **derecognized** if they act against the pledge or against any of the requirements attached to their type.
```
